Jenkinsfile

```diff
@@ -45,10 +45,14 @@ try {
   }
 
   node('linux') {
+    stage("hostname") {
+      sh "hostname"
```

Comment:
"Print the hostname so that we can reproduce what node the docker image was executed on"